resources/multisite_drupal_standard.make

```diff
@@ -153,6 +153,12 @@ projects[css_injector][patch][] = https://www.drupal.org/files/issues/css_inject
 
 projects[ctools][subdir] = "contrib"
 projects[ctools][version] = "1.9"
+; Deprecate PHP 4 style constructors for PHP 7 compatibility
+; https://www.drupal.org/node/2528736
+; The patch corresponds to commit 96e56f523ce3d9609e677fcc6263cfc3e798b0ab in
+; the ctools repository.
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-11467
+projects[ctools][patch][] = patches/ctools-php7-2528736.patch
```

Comment:
@dxvargas, the patch provided in the ticket was not exactly what was committed to the ctools repository. In order to have what will be released in future ctools version I've used the committed code instead.

I will adjust the patch to also include the fix for https://www.drupal.org/node/2657834.

Please note you are reviewing code again which is actually part of https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-165. I had to base this feature branch on that other feature branch because otherwise we would lack some patches already.